src/jdk.compiler/share/classes/com/sun/tools/javac/parser/JavaTokenizer.java

```diff
@@ -1537,6 +1549,50 @@ UnicodeReader trimJavadocComment(UnicodeReader line) {
             return line;
         }
 
+        /**
+         * Determine how much indent to remove from markdown comment.
+         *
+         * @return minimum indent to remove
+         */
+        int getMarkdownIndent() {
+            int result = Integer.MAX_VALUE;
+            UnicodeReader fullReader = lineReader(position(), position() + length());
+
+            while (fullReader.isAvailable()) {
+                UnicodeReader line = fullReader.lineReader();
+                line.skipWhitespace();
+                line.accept("///");
+                int pos = line.position();
+                line.skipWhitespace();
+
+                if (line.isAvailable()) {
+                    result = Integer.min(result, line.position() - pos);
+                }
+            }
+
+            return result == Integer.MAX_VALUE ? 0 : result;
+        }
+
+        /**
+         * Trim the first part of the Markdown comment.
+         *
+         * @param indent how much indentation to remove
+         * @param line line reader
+         *
+         * @return modified line reader
+         */
+        UnicodeReader trimMarkdownComment(UnicodeReader line, int indent) {
+            int pos = line.position();
```

Comment:
Unused.

Reply:
Yes, and the method has no side effects. Removed.